keras_nlp/tokenizers/word_piece_tokenizer_test.py

```diff
@@ -170,6 +172,41 @@ def test_from_file(self):
         call_output = tokenizer(input_data)
         self.assertAllEqual(call_output, [[1, 2, 3, 4, 5, 6, 7]])
 
+    def test_from_pretrained(self):
```

Comment:
Are you downloading files here as part of the unit test? We might need to mock this instead but remote calls are generally discouraged for unit tests and might break some automated testing systems.

Reply:
Discussed offline with @fchollet. Let's omit all tests that make RPCs.

Reply:
sg! Will remove any test that requires network

Reply:
Here's my proposal for model loading testing: https://github.com/keras-team/keras-nlp/issues/302